package.json

```diff
@@ -37,13 +37,17 @@
     "flagged-respawn": "^1.0.1",
     "is-ci": "^2.0.0",
     "jest-environment-node": "^25.1.0",
+    "mixpanel": "^0.11.0",
     "ncp": "^2.0.0",
     "near-api-js": "^0.23.1",
     "open": "^7.0.1",
+    "os": "^0.1.1",
```

Comment:
why `os` and `path` added as external deps? There should be built-in modules normally.

Reply:
thanks! Didn't know they were built-in. Removed from package.json